x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/gen/script/Scripts.java

```diff
@@ -77,6 +78,15 @@ public static ScriptTemplate nullSafeFilter(ScriptTemplate script) {
                 DataTypes.BOOLEAN);
     }
 
+    public static ScriptTemplate nullsSort(ScriptTemplate script, Sort.Missing missing) {
+        int nullOrd = missing == Sort.Missing.FIRST ? -1 : 1;
+        return new ScriptTemplate(
+            format(Locale.ROOT, "(%s) == null ? %d : %d", script.template(), nullOrd, nullOrd * -1),
```

Comment:
I'm somewhat unsure about using the if operator here versus introducing dedicated painless methods. The big advantage of this expression is that it does not introduce bwc issues.

I've also looked into repurposing existing methods (`isNull` and `isNotNull`) but they are not applicable here because painless does not implicitly cast `Boolean` to `double`.